indico/modules/events/registration/client/js/form/fields/SingleChoiceInput.jsx

```diff
@@ -267,20 +239,53 @@ export default function SingleChoiceInput({
   return component;
 }
 
+SingleChoiceInputComponent.propTypes = {
+  disabled: PropTypes.bool.isRequired,
+  isRequired: PropTypes.bool.isRequired,
+  itemType: PropTypes.oneOf(['dropdown', 'radiogroup']).isRequired,
+  choices: PropTypes.arrayOf(PropTypes.shape(choiceShape)).isRequired,
+  withExtraSlots: PropTypes.bool.isRequired,
+  // TODO: placesUsed, captions - only needed once we deal with real data
+};
+
+export default function SingleChoiceInput({
+  htmlName,
+  disabled,
+  isRequired,
+  defaultValue,
```

Comment:
This isn't correct; there's a default item not a default value. I've pushed a fix for this component to your branch, but please check whether e.g. the multi choice or accommodation field have the same bug.